src/runtime/opencl.cpp

```diff
@@ -38,6 +38,7 @@ extern "C" WEAK void *halide_opencl_get_symbol(void *user_context, const char *n
         "opencl.dll",
 #else
         "libOpenCL.so",
+        "libOpenCL.so.1",
```

Comment:
I wonder if it's the case that `libOpenCL.so.1` should rather _replace_ `libOpenCL.so`? The latter is a namelink that's only present when `-dev` packages are installed. It should always point to the former.

Reply:
I was thinking the same. Can fix it later, but I needed this on my local machine, so without being to destructive without consensus, I did this.

Reply:
I removed it. We'll see what the build bots do.